ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/tls/SecretsUrl.scala

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.tls
+
+import java.io.InputStream
+import java.net.URL
+import java.nio.file.Path
+
+// This trait is not sealed so we can replace it with a fake in tests.
+trait SecretsUrl {
```

Comment:
What's about secrets in this class?

Reply:
Nothing, it's just how it's used.

Happy to bikeshed further on this but in a separate changeset, as this is causing failing builds in CI.